.env.test

```diff
@@ -0,0 +1,17 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=test
+FLASK_SKIP_DOTENV=True
+
+# Change the following values for local development.
+SECRET_KEY=test_secret_key
+
+# TEST Trello API Details
+TRELLO_API_KEY=this_is_trello_api_key_123
+TRELLO_TOKEN=and_this_is_the_trello_token_56789
+BOARD_ID=5fb51906cc7d395b1b145b0e
```

Comment:
Well remembered to include this update, it would be nice to also mention all of the new variables in the `.env.template` file

Reply:
This has now been done